packages/dcos-checks/extra/dcos-checks.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
```

Comment:
Can we call this `dcos-checks-poststart.service`? We'll need to distinguish this unit from the prestart one once that's added.